site/styles/_footer.scss

```diff
@@ -94,4 +94,27 @@ footer {
 			text-decoration: none;
 		}
 	}
+}
+// Festival Scope
+.footer-festivalscope {
+  font-size: .8rem;
+
+  a {
+    padding: 3px 0 6px 0;
+    margin: 0;
+
+    &:hover .footer-festivalscope-logo {
+      filter: invert(100%);
```

Comment:
Not sure about this though

Reply:
Not sure in what way?

Reply:
Ignore... wasn't sure how it would look in practice and in particular how the inversion looked on different client colours, but happy enough that will have to be tweaked for each client rather than a silver bullet here.